tileserver/__init__.py

```diff
@@ -433,7 +434,13 @@ def read_tile(self, coord, offset=None):
         else:
             fmt = json_format
 
-        raw_data = self.store.read_tile(coord, fmt, 'all')
+        try:
+            raw_data = self.store.read_tile(coord, fmt, 'all')
+        except BotoServerError:
```

Comment:
Do we want to catch the default / blank set of exceptions (i.e: `except:` - I don't know if that has a name)? Just in case there are other kinds of error (network timeouts, DNS failures) that we can cover at the same time?